source/includes/api/_query.md

```diff
@@ -111,33 +349,21 @@ SELECT ST_HISTOGRAM(rast, 1, auto, true) FROM table
 SELECT ST_valueCount(rast, 1, true) FROM table
 ```
 
-## Freeze query
-
-It is possible generate a json file in a bucket of the sql result. You only need send a query param freeze with value true and you will obtain the url of the json file.
-
-<aside class="notice">
-    This is an authenticated feature!
-</aside>
-
-```shell
-curl -i -XGET http\://api.resourcewatch.org/v1/query\?sql\=SELECT\ \*\ from\ <dataset.slug>&freeze=true
-```
-
-## Rasdaman queries
+### Rasdaman queries
 
 SQL-like queries can be employed for accessing data stored in Rasdaman datasets. Subsets on the original axes of the data may be provided in the WHERE statement. So far, only operations that result in a single scalar can be obtained from Rasdaman - averages, minimums, maximums.
 
 ```shell
 curl -XGET https://api.resourcewatch.org/v1/query?sql=SELECT avg(Green) from 18c0b71d-2f55-4a45-9e5b-c35db3ebfe94 where Lat > 0 and  Lat < 45 \
-	-H 'Content-Type: application/json' \
-	-H 'Authorization: Bearer <token>'
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: Bearer <token>'
 ```
 
-## NEX-GDDP queries
+### NEX-GDDP queries
```

Comment:
being this a dataset provider, like carto or document, should it really be inside "document-based datasets"?

Reply:
Yep, agree, moving it!